software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcess.java

```diff
@@ -29,6 +30,7 @@
 
 import com.google.common.collect.ImmutableSet;
 
+@Catalog(name="Vanilla Windows Process", description="A basic windows entity configured with scripts, e.g. for launch, check-running and stop")
```

Comment:
A really trivial request but could you make it `Windows` instead of `windows` in the description? As it's in a prominent location - and a trademarked description - we should be accurate. Thanks!